istio/datadog_checks/istio/istio.py

```diff
@@ -64,9 +65,17 @@ def check(self, instance):
             # Process process_galley
             self.process(process_galley_config)
 
+        # Get the config for the process_citadel instance
+        process_citadel_endpoint = instance.get('citadel_endpoint')
+        if process_citadel_endpoint:
+            process_citadel_config = self.config_map[process_citadel_endpoint]
+
+            # Process process_citadel
+            self.process(process_citadel_config)
+
         # Check that at least 1 endpoint is configured
-        if not (process_galley_endpoint or process_pilot_endpoint or process_mixer_endpoint or istio_mesh_endpoint):
-            raise CheckException("At least one of Mixer, Mesh, Pilot, or Galley endpoints must be configured")
+        if not (process_galley_endpoint or process_pilot_endpoint or process_mixer_endpoint or istio_mesh_endpoint or process_citadel_config):
```

Comment:
Starts to become a long line, if you're willing to refactor we could have something either like `any(endpoint1, endpoint2...)` which could be a little more readable, or a `has_processed_something` boolean variable that we can set to True if any of the previous if evaluates to True.

Reply:
Done.